packages/signature-controller/src/SignatureController.ts

```diff
@@ -133,7 +128,11 @@ export class SignatureController extends BaseControllerV2<
 > {
   hub: EventEmitter;
 
-  #keyringController: KeyringController;
+  readonly #keyringControllerSignMessage: KeyringController['signMessage'];
+
+  readonly #keyringControllerSignPersonalMessage: KeyringController['signPersonalMessage'];
+
+  readonly #keyringControllerSignTypedMessage: KeyringController['signTypedMessage'];
```

Comment:
Suggestions for a better naming that does not collide with internal #signMessage, #signPersonalMessage and #signTypedMessage are well accepted 😄